convert2rhel/utils.py

```diff
@@ -274,6 +274,18 @@ def get_traceback_str():
     return "".join(traceback.format_exception(exc_type, exc_value, exc_traceback))
 
 
+def remove_tmp_dir():
+    """Remove temporary folder (TMP_DIR), not needed post-conversion."""
+    try:
+        shutil.rmtree(TMP_DIR)
+        loggerinst.info("Temporary folder %s removed" % TMP_DIR)
+    except OSError as err:
+        loggerinst.info("Failed removing temporary folder %s" % TMP_DIR)
+        loggerinst.info("Error (%s): %s" % (err.errno, err.strerror))
+    except Exception:
+        loggerinst.info("Other error while removing temporary folder %s" % TMP_DIR)
```

Comment:
```suggestion
```
I don't expect other than OSError tree exceptions. And this message would not tell much anyway.

Reply:
Totally agree, moreover bare exception is antipattern in python. The reason is that you need to be explicit about the program behaviour, otherwise you can miss some important exception from your radar

Reply:
@ZhukovGreen @bocekm Sorry guys, this isn't actual version. As I had problems with the `tmpdir` I didn't update the branch online. 

For the exceptions: now I'm catching `TypeError` due `None` value of `TMP_DIR`. I'll update the branch when I finish the integration test